modules/tinyexr/image_loader_tinyexr.cpp

```diff
@@ -131,21 +133,34 @@ Error ImageLoaderTinyEXR::load_image(Ref<Image> p_image, FileAccess *f, bool p_f
 			return ERR_FILE_CORRUPT;
 		}
 	}
-
+*/
```

Comment:
Shouldn't the code be removed instead of commented out?

Reply:
Well the support for single-channel grayscale EXR should be preserved though, some users rely on it (e.g. @Zylann IIRC).

Reply:
Removed it